corehq/apps/locations/tests/test_location_expressions.py

```diff
@@ -69,44 +69,75 @@ def test_location_not_found(self):
         self._check_expression(doc, None)
 
 
-class TestLocationParentIdExpression(TestCase):
+class LocationHierarchyTest(TestCase):
```

Comment:
There's also a `LocationHierarchyTestCase` (and associated `setup_locations_and_types`) that's pretty similar.